### PR TITLE
feature/SM-3969: marking an asd as provided

### PR DESCRIPTION
**Pull request by ccurran01**
Screenshot 2020-05-19 at 17 17 01

### CODE REVIEW

File: docs/api-documentation/V1.23/index.md [hunk 1]
```diff
@@ -2817,6 +2817,7 @@ Updated Works API with the following changes:
       <li><code>PermitResponse</code></li>
       <li><code>WorkCreateRequest</code></li>
     </ol>
+    While these fields are optional and requests will succeed without providing them, the Work API does compare the ASDs submitted in the request with ASDs returned by the Street Lookup API for the given USRN. This is to determine whether an ASD was provided. In order for the ASD to be correctly flagged as provided, where these properties were returned by the street lookup API, these fields will need to match the Lookup API response.
```
[the remark this answers is not in the document]
Comment (ccurran01) [reply docs/api-documentation/V1.23/index.md:2820]: We were deliberately putting this in the previous version to give API developers some warning that this change was coming. We probably should have documented these changes last sprint. Steven I believe was to put a message in the DfT slack about this update to this version of the documentation. Here we were following up on https://github.com/departmentfortransport/street-manager-docs/pull/469 to just give a bit more detail about how to use these. 

Might be worth making adding something into v1.24 as well though

[the remark this answers is not in the document]
Comment (cpaul92) [reply docs/api-documentation/V1.23/index.md:2820]: Yeah I think it is goes into a version it should be in any later version so should probably be added to v1.24 for consistency 👍

[the remark this answers is not in the document]
Comment (ccurran01) [reply docs/api-documentation/V1.23/index.md:2820]: Sure 👍

File: docs/api-documentation/V1.24/index.md [hunk 2]
```diff
@@ -2804,6 +2804,21 @@ Updated Works API with the following changes:
   <li>
     The <code>InspectionResponse</code> and <code>InspectionSummaryResponse</code> response models have been updated to make the <code>inspection_category</code> property optional.
   </li>
+  <li>
+    The definitions for <code>ASDCode</code> and <code>ASDPeriodicityCode</code> models have been updated to clarify that the enum values should be submitted as numbers instead of strings. <strong>Note that, in a future release, we will be adding validation against providing strings.</strong> The API will continue to accept both for the time being, but we strongly encourage submitting these values as numbers instead of strings if you aren't already doing so.
+  </li>
+  <li>
+    <code>PermitASD</code> interface, which is used in a number of work and permit request and response models, will be updated to include new optional date properties <code>special_desig_start_date</code> and <code>special_desig_end_date</code>. These fields will be stored against the work and returned on the appropriate responses where avilable. Affected models include:
```
Comment (matthewdoreilly) [inline docs/api-documentation/V1.24/index.md:2811]:
```suggestion
    <code>PermitASD</code> interface, which is used in a number of work and permit request and response models, will be updated to include new optional date properties <code>special_desig_start_date</code> and <code>special_desig_end_date</code>. These fields will be stored against the work and returned on the appropriate responses where available. Affected models include:
```

**Pull request merged**